doc/interface/spatial_discretization_methods.rst

```diff
@@ -0,0 +1,127 @@
+For the CADET-DG scheme, it is :math:`N_d + 1` with :math:`N_d` denoting the polynomial degree, and it can thus be user-defined by specifying the field ``POLYDEG`` (and ``PAR_POLYDEG`` for the GRM).
+As a convergence order of > 6 is hardly realized within the approximation error of engineering tolerance, we recommend a maximum polynomial order of 5.
+As the FV scheme oftentimes yields an EOC of around 2.5 and is computationally more enhanced (less arithmetic operations per DOF and customized factorization) than the DG code, we recommend a polynomial degree of at least 3 to top this.
+
+Smooth solutions
+----------------
+
+Smoothness in functions is characterized by the absence of sudden changes, reflected in the continuity and differentiability of the function and its derivatives.
+In numerical simulation, smoothness indicators are often based on derivatives or moments of the solution.
+That is, strong gradients and high frequencies are used to identify non-smooth parts of the solution.
+Godunov's order barrier theorem shows why the concept of smoothness plays a crucial role in the deployment of numerical methods.
+It states that linear high-order methods that are monotonous are at most first-order accurate.
+Linear higher-order (:math:`> 1`) methods thus suffer from artificial oscillations at non-smooth parts of the solution, specifically at discontinuities and strong gradients.
+Some higher-order methods, such as CADET-FV (2nd order), contain a non-linear mechanism to suppress these oscillations.
+The non-linear WENO mechanism employed in CADET-FV can be fine-tuned via the fields specified here :ref:`flux_restruction_methods`.
+Unfortunately, non-linear higher-order methods (order :math:`\geq 3`) are either not applicable (e.g., undefined boundary treatment) or have other shortcomings, such as more highly problem-dependent parameters.
+
+CADET-DG is a linear high-order method (arbitrary order) and thus exhibits oscillatory behaviour at strong gradients, which increases the approximation error and results in a smaller EOC for lower resolutions.
+As strong gradients are a local phenomena that can be captured by employing more discrete points, DG becomes more performant again for higher resolutions.
+This, however, might happen after the engineering error tolerance is by far surpassed.
+Hence, CADET-FV as a stabilized lower-order method can be more performant, depending on the setting.
+The DG scheme reduces its oscillatory behaviour by adding artificial numerical dispersion at element interfaces.
+Thus, the use of a lower polynomial degree and more elements is recommended for rather non-smooth problems.
+
+In Chromatography, mathematical discontinuities never happen, as there are always some dispersive effects in reality.
+Chromatography models, however, allow for discontinuities if dispersion parameters are set to zero.
+Moreover, steep and self-sharpening concentration fronts might appear due to binding and steep injection profiles.
```

Comment:
Only focus on the self-sharpening here. The steep injection profiles are covered by the sentence before.